src/plugins/intel_npu/src/plugin/npuw/lazy_tensor.cpp

```diff
@@ -33,6 +33,11 @@ Const::Const(const std::shared_ptr<ov::op::v0::Constant>& n) : m_node(n) {
     auto weightless_cache_attr = rt_info.find(ov::WeightlessCacheAttribute::get_type_info_static());
     if (weightless_cache_attr != rt_info.end()) {
         m_offset = weightless_cache_attr->second.as<ov::WeightlessCacheAttribute>().bin_offset;
+    } else {
```

Comment:
so weightless cache attribute gets attached before our rope-cache pass - right?

Reply:
Yes, it's assigned for all Constants in the original IR. New constants don't have it by default